examples/SED_BANDPASS_examples/example_SedBandpass_galaxy.py

```diff
@@ -87,7 +89,7 @@
 
 
 # show results
-print "#sedname      fluxnorm     redshift  ebv_gal   ebv_mw  magnitude "
+print("#sedname      fluxnorm     redshift  ebv_gal   ebv_mw  magnitude ")
 for i in range(len(galaxykeys)):
-    print "%s %.5g  %.3f %.5f %.5f %.5f" %(galaxykeys[i], fluxnorm[i], redshifts[i], ebv_gal[i], ebv_mw[i], mags[i])
+    print("%s %.5g  %.3f %.5f %.5f %.5f" %(galaxykeys[i], fluxnorm[i], redshifts[i], ebv_gal[i], ebv_mw[i], mags[i]))
     
```

Comment:
This script doesn't run anyway on account of
```
OSError: The throughput file exampleBandpass.dat does not exist
```
I would rather not touch these example scripts any more than I have to given that they have undergone serious bit rot.